prime-dotnet-webapi/PrimeConstants.cs

```diff
@@ -21,5 +21,8 @@ public static class PrimeConstants
         public readonly static string PHARMANET_API_PASSWORD = Environment.GetEnvironmentVariable("PHARMANET_API_PASSWORD") ?? "";
         public readonly static string PHARMANET_SSL_CERT_FILENAME = Environment.GetEnvironmentVariable("PHARMANET_SSL_CERT_FILENAME") ?? "";
         public readonly static string PHARMANET_SSL_CERT_PASSWORD = Environment.GetEnvironmentVariable("PHARMANET_SSL_CERT_PASSWORD") ?? "";
+
+        // Logging
+        public readonly static string LOG_FILE_PATH = Environment.GetEnvironmentVariable("LOG_FILE_PATH") ?? "logs";
```

Comment:
@poggdogg - heads up on this. We'll need something to watch this. Also, if we have multiple pods should this logfile path be a shared mounted volume?

Reply:
Shared, mounted volume sounds right for now - at least until we get a logging spike done.  That can be done later though and I can hijack the existing log file directory.  This shouldn't require any code changes.